code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -74,12 +80,6 @@
 	singular_name = "grass floor tile"
 	desc = "A patch of grass like they often use on golf courses."
 	icon_state = "tile_grass"
-	w_class = 3.0
-	force = 1.0
-	throwforce = 1.0
-	throw_speed = 3
-	throw_range = 7
-	max_amount = 60

```

Comment:
you can also remove these from /obj/item/stack/tile/plasteel in  code/game/objects/itemsstacks/tiles/plasteel.dm , or maybe even move plasteel tile in this file since there's no reason to have two files.